### PR TITLE
feat(Rows, Cards, FeedbackScreen, FormFields, Buttons): add test ids for components and their internal elements

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-2080)

### CODE REVIEW

File: src/boxed.tsx [hunk 1]
```diff
@@ -90,6 +90,7 @@ export const InternalBoxed = React.forwardRef<HTMLDivElement, Props & InternalPr
         ref
     ) => {
         const isInverseOutside = useIsInverseOrMediaVariant();
+        const isInverseInside = isInverse || variant === 'inverse' || variant === 'media';
```
Comment (marcoskolodny) [inline src/boxed.tsx:93]: Unrelated, but Boxed was not working properly when using media variant

File: src/card.tsx [hunk 2]
```diff
@@ -1023,13 +1051,18 @@ export const SnapCard = React.forwardRef<HTMLDivElement, SnapCardProps>(
...
         return (
             <CardContainer
-                dataAttributes={{'component-name': 'SnapCard', ...dataAttributes}}
+                dataAttributes={{'component-name': 'SnapCard', testid: 'SnapCard', ...dataAttributes}}
                 ref={ref}
                 className={styles.touchableContainer}
                 aspectRatio={aspectRatio}
                 aria-label={isTouchable ? undefined : ariaLabelProp}
             >
-                <Boxed className={styles.boxed} isInverse={isInverse} width="100%" minHeight="100%">
```
Comment (marcoskolodny) [inline src/card.tsx:1032]: unrelated, but I saw this usages of deprecated isInverse prop and updated them in order to use variant instead

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Review by bnombela — APPROVED** [review]
Marvelous!

**Pull request merged**